config/routes/claims.rb

```diff
@@ -1,4 +1,6 @@
-scope module: :claims, as: :claims, constraints: { host: ENV["CLAIMS_HOST"] } do
+scope module: :claims, as: :claims, constraints: {
+  host: [ENV["CLAIMS_HOST"], *ENV.fetch("CLAIMS_HOST", "").split(",")],
```

Comment:
Isn't this suppose to be like?
```suggestion
  host: [ENV["CLAIMS_HOST"], *ENV.fetch("CLAIMS_HOSTS", "").split(",")],
```

Reply:
Yes... 🙈